web-wallet/src/lib/stores/networkStore.js

```diff
@@ -1,8 +1,22 @@
 import { writable } from "svelte/store";
+import { browser } from "$app/environment";
 import { AddressSyncer, Network } from "$lib/vendor/w3sper.js/src/mod";
+import { makeNodeUrl } from "$lib/url";
+
+function getNetworkUrl() {
+  if (browser) {
+    return makeNodeUrl();
+  } else {
+    return (
+      (import.meta.env.VITE_NODE_URL &&
+        new URL(import.meta.env.VITE_NODE_URL)) ||
+      "https://localhost"
+    );
+  }
+}
```

Comment:
You can get rid of this function and of the `browser` import by giving a value to the env variable in tests.

This function has no meaning for the client code and exists only to accommodate to tests: it should be the other way around.

Reply:
I'm not following... This code isn't for the tests, it's because the networkStore is called in SSR (dev mode).